src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -156,23 +120,24 @@ public void update(StandaloneConfig standaloneConfig) {
             aliases.add(author.getGitId());
             aliases.forEach(alias -> newAuthorAliasMap.put(alias, author));
         }
-        validateFormats(standaloneConfig.getFormats());
-        validateIgnoreCommits(standaloneConfig.getIgnoreCommitList());
+
+        Format.validateFormats(standaloneConfig.getFormats());
+        Commit.validateCommits(standaloneConfig.getIgnoreCommitList());
```

Comment:
Was thinking we no longer need this `validateFormats`/`validateCommits` method anymore, since when we convert the `String` -> `Commit`/`Format` we will do the validation check in the constructor itself.

However, since our current behaviour is that we completely ignore the `StandaloneConfig` if there are any invalid values inside, we can leave this here for now. Might want to remove this in the future, and allow using of the `StandaloneConfig` with only the valid values inside.